index.html

```diff
@@ -319,8 +319,331 @@ <h4>Definitions</h4>
       </table>
     </section>
 
+    <section>
+      <h2><dfn data-lt="controller" data-dfn-type="dfn" id="controller">controller</dfn></h2>
+      <a href="https://www.w3.org/TR/did-core/#dfn-did-controllers">DID Controller.</a>
+
+      <pre class="example">
+        {
+          "controller": "did:example:123",
+          ...
+        }
+      </pre>
+
+      <h4>Definitions</h4>
+      <table class="simple" style="width: 100%;">
+        <thead>
+          <tr>
+            <th>JSON Schema</th>
+            <th>JSON-LD</th>
+            <th>CBOR</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="https://w3c.github.io/did-core-registries/schemas/didDocument/didDocument.json">JSON Schema</a>
+            </td>
+            <td>
+              <a href="https://w3c.github.io/did-core-registries/contexts/did-v1.jsonld">JSON LD</a>
```

Comment:
The whole doc is using "JSON LD" rather than "JSON-LD".  Was there a reason or is that just getting copied around?

Reply:
its just a paste error. I will update it to "JSON-LD".